pkg/k8shandler/fluentd.go

```diff
@@ -276,7 +276,7 @@ func newFluentdPodSpec(cluster *logging.ClusterLogging, elasticsearchAppName str
 			},
 		}
 	}
-	fluentdContainer := NewContainer("fluentd", "fluentd", v1.PullIfNotPresent, *resources)
+	fluentdContainer := NewContainer("fluentd", "fluentd", v1.PullAlways, *resources)
```

Comment:
Keeping `v1.PullIfNotPresent` is the correct value? Probably a miss from testing this PR.

Reply:
yes, a leftover from the testing.

Reply:
but it is a pain to change it everytime you test with changes in fluent image.
k8s will always pull the image if we dont specify the pull policy, and the image tag is `latest`.
In production, we will use sha based tags, so image will be pulled only once if we leave it unspecified.
So perhaps we should consider removing the pull policy.